docs/development/contributing_guide.md

```diff
@@ -53,6 +53,13 @@ can find many good git tutorials on the web.
 
 # 4. Install the dependencies
 
+Before installing the Python dependencies, make sure you have installed Rust (see the
+"What do I need?" section above) and have selected the default Rust toolchain:
```

Comment:
> and have selected the default Rust toolchain

This doesn't seem strictly necessary? I'm assuming the goal is just to suggest an easy way to install the latest stable version of Rust. We can probably just re-iterate what the section above explains.


```suggestion
Before installing the Python dependencies, make sure you have installed a recent version
of Rust (see the "What do I need?" section above). The easiest way of installing the
latest version is to use [rustup](https://rustup.rs/).:
```

Reply:
I added the comment about the default rust toolchain because after installing Rust via rustup and then trying a `poetry build` I got this error:

```
running build_ext
running build_rust
error: rustup could not choose a version of rustc to run, because one wasn't specified explicitly, and no default is configured.
help: run 'rustup default stable' to download the latest stable release of Rust and set it as your default toolchain.
error: can't find Rust compiler
```
So that was cargo-culting on my part - I have no experience with Rust, I just did what the error message suggested and then a build succeeded.

Reply:
OK, I removed rust completely from my system and tried everything from scratch - I could not reproduce the error about the default rust toolchain, so I've removed the mention.